### ISSUE TITLE
image not Available

### ISSUE THREAD

**Issue by doczong** [ufodone/envisalink_new#131]
Running HA 2024.7.4

Broken image in device add, no image at top of device config and broken image icon beside device in details.

Just confirming this isn't just my install.  It otherwise works, just killing my OCD. :)

**Comment by ufodone** [comment]
Yes, that's a known issue.  I had plans to rename this integration so it would be easier to use as a drop-in replacement (which would in turn fix the image issue).  But I haven't gotten around to that so I'll maybe look to get new images submitted for `envisalink_new`.

**Comment by ufodone** [comment]
Images should be showing up now.
